### DISCUSSION TITLE
Group/Ungroup actions don't issue expected signals

### DISCUSSION THREAD

**Discussion by Elgato-AStory** [obsproject/obs-studio#6251] [Development Questions] [answered]
The signals emitted when grouping existing sources directly (selecting the items, right clicking, and selecting "Group Selected Items") and ungrouping do not include the `item_add` and `item_remove` signals for the scene item associated with the group. Is this an intentional design choice? It strikes me as a bit awkward that they are fired when creating a group via "Add > Group" and removing the group with "Remove" but not when you select "Group Selected Items" and "Ungroup."

**Comment by tt2468** [comment, marked as answer]
This bug has been on my list for a while, it'll probably be fixed by the time 28 comes around.
